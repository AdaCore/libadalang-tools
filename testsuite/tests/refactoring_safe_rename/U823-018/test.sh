# Rename the generic package declaration
safe_rename -P default.gpr -S gen.ads -L 3 -R 10 -N Genn --algorithm=analyse_ast
# Rename a declaration of a generic package declaration
safe_rename -P default.gpr -S gen.ads -L 4 -R 4 -N Constt --algorithm=analyse_ast
