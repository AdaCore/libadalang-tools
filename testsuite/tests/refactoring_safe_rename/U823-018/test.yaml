description: |
    This test checks if the safe rename tool correctly handles
    the rename of a generic package declaration and declarations
    inside it.

driver: shell_script
