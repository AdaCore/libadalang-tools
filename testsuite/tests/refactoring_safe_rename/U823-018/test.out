# Renaming node gen.ads:3:9: 
References:
Unit:gen.ads
Line: 3
Source Location Range:
Start_Line: 3
Start_Column: 9
End_Line: 3
End_Column: 12
Line: 5
Source Location Range:
Start_Line: 5
Start_Column: 5
End_Line: 5
End_Column: 8
Unit:main.adb
Line: 2
Source Location Range:
Start_Line: 2
Start_Column: 6
End_Line: 2
End_Column: 9
Line: 4
Source Location Range:
Start_Line: 4
Start_Column: 27
End_Line: 4
End_Column: 30
File renames:
gen.ads -> genn.ads

# Renaming node gen.ads:4:4: 
References:
Unit:gen.ads
Line: 4
Source Location Range:
Start_Line: 4
Start_Column: 4
End_Line: 4
End_Column: 9
Unit:main.adb
Line: 6
Source Location Range:
Start_Line: 6
Start_Column: 34
End_Line: 6
End_Column: 39

